tools/sklearn/train_test_split.xml

```diff
@@ -0,0 +1,115 @@
+<tool id="sklearn_train_test_split" name="Split Dataset" version="@VERSION@">
+    <description>into train and test subsets</description>
+    <macros>
+        <import>main_macros.xml</import>
+    </macros>
+    <expand macro="python_requirements"/>
+    <expand macro="macro_stdio"/>
+    <version_command>echo "@VERSION@"</version_command>
+    <command detect_errors="exit_code"><![CDATA[
+        python '$__tool_directory__/train_test_split.py'
+            --inputs '$inputs'
+            --infile_array '$infile_array'
+            #if $options.shuffle_selection.shuffle not in ['None', 'simple']
+            --infile_labels '$options.shuffle_selection.labels'
+            #end if
+            --outfile_train '$out_train'
+            --outfile_test '$out_test'
+    ]]>
+    </command>
+    <configfiles>
+        <inputs name="inputs"/>
+    </configfiles>
+    <inputs>
+        <param name="infile_array" type="data" format="tabular" label="Select the dataset containing array to split"/>
+        <param name="infile_info" type="select" multiple="true" display="checkboxes" optional="true" label="The dataset">
+            <option value="header" selected="true">Has headers</option>
+        </param>
+        <section name="options" title="Options" expanded="true">
+            <param argument="test_size" type="float" optional="False" value="0.25" label="Test size:"
+                help="If float, should be between 0.0 and 1.0 and represent the proportion of the dataset to include in the test split. If int, represents the absolute number of test samples."/>
```

Comment:
I think this is not very intuitive. Could you create an option to allow the user to choose to enter either an absolute value or a proportion?

Reply:
I agree with you. yet, do you notice that scikit-learn sets the parameter like that? Maybe most people just stick to the proportion one.